src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -642,7 +642,8 @@ private void startWatchServiceOnLogFilePaths() throws IOException, InterruptedEx
             try {
                 componentLogConfigurations.forEach((componentName, componentLogConfiguration) -> {
                     // Only register the path of a component if the disk space limit is set.
-                    if (componentLogConfiguration != null && componentLogConfiguration.getDiskSpaceLimit() > 0) {
+                    if (componentLogConfiguration != null && componentLogConfiguration.getDiskSpaceLimit() != null
```

Comment:
Do we let `.getDiskSpaceLimit()` return `null` on purpose? Is there a reason not asking it to return a default value if not it's not configured? (Don't know if it's a good pattern though).

Reply:
If we had a default that's what we'd do but we don't have a default